packages/@ama-sdk/schematics/schematics/code-generator/open-api-cli-generator/open-api-cli.generator.ts

```diff
@@ -65,7 +75,9 @@ export class OpenApiCliGenerator extends CodeGenerator<OpenApiCliOptions> {
 
   /** @inheritDoc */
   protected runCodeGeneratorFactory = (factoryOptions: { rootDirectory?: string } = {}) => {
-    const rootDirectory = factoryOptions.rootDirectory || process.cwd();
+    const rootDirectory = factoryOptions.rootDirectory ?
+      (path.isAbsolute(factoryOptions.rootDirectory) ? factoryOptions.rootDirectory : path.resolve(process.cwd(), factoryOptions.rootDirectory)) :
```

Comment:
`path.resolve` or `path.posix.resolve` ?
When should we use `posix` and when not ?

Reply:
my understanding is that we need to use `posix` at least when building paths for the Tree of schematics and for globby
when targeting an actual on the disk we should use `path.resolve` to match the OS paths

also, for the tree of schematics we should not use `resolve` as it create an absolute path

Reply:
This is for the `cwd` option of `spawn` function, so it should be aligned with OS path separator.
Globally par default it is **not** `posix` except when we want a path to print in a `json` file (cd tsconfig.json, angular.json, etc...) or use it in `tree` or `require` paths